editor/src/setupTests.js

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
```

Comment:
In a project as timeboxed as this one, testing is an ambitious goal. I would strip out jest and the related files if you're not running any tests.